### PR TITLE
update translations for 4.0 release

### PR DESCRIPTION
**Pull request by maxme**

### CODE REVIEW

**Comment by maxme** [comment]
@tonyr59h I introduced an issue in my previous script update (en-rUS was not in the language selector anymore), added back in https://github.com/wordpress-mobile/WordPress-Android/commit/5f6460bffeca58f59ee866cfe505a7bc4b569735

File: WordPress/src/main/res/values/available_languages.xml [hunk 1]
```diff
@@ -130,6 +126,22 @@ mk
 en-au
 </item>
 
+<item>
+sr
```
Comment (tonyr59h) [inline WordPress/src/main/res/values/available_languages.xml]: Good to see the new languages in!

**Comment by tonyr59h** [comment]
> en-rUS was not in the language selector anymore

I'm used to selecting device default at the top of the list for English so I'm not on the lookout for English (US). I will be now though, good catch!

**Comment by tonyr59h** [comment]
:shipit:

**Pull request merged**